wheaton-college-phd-in-biblical-and-theological-studies.csl

```diff
@@ -331,8 +333,10 @@
         <text term="in" text-case="capitalize-first" suffix=" "/>
         <choose>
           <if variable="volume" match="any">
-            <text term="volume" form="short" suffix=" "/>
-            <text variable="volume" prefix=" " suffix=" of "/>
+            <group delimiter=" ">
+              <text term="volume" form="short"/>
+              <text variable="volume" suffix=" of "/>
```

Comment:
@dowens76 -- could you take a quick look at this? This looks like number-of-volumes is missing here?